README.md

```diff
@@ -25,7 +25,9 @@ Litmus takes a cloud-native approach to create, manage and monitor chaos. Chaos
 - **ChaosExperiment**: A resource to group the configuration parameters of a chaos experiment. ChaosExperiment CRs are created by the operator when experiments are invoked by ChaosEngine.
 - **ChaosResult**: A resource to hold the results of a chaos-experiment. The Chaos-exporter reads the results and exports the metrics into a configured Prometheus server.
 
-Chaos experiments are hosted on <a href="https://hub.litmuschaos.io" target="_blank">hub.litmuschaos.io</a>. It is a central hub where the application developers or vendors share their chaos experiments so that their users can use them to increase the resilience of the applications in production.
+Chaos experiments are hosted on <a href="https://
+
+" target="_blank">hub.litmus chaos.io</a>. It is a central hub where the application developers or vendors share their chaos experiments so that their users can use them to increase the resilience of the applications in production.
```

Comment:
There is an empty new line which breaks the link in the anchor tag, besides this change doesn't really modify/update any content here.